mpqp/core/instruction/gates/native_gates.py

```diff
@@ -177,6 +237,31 @@ def to_other_language(
                     "export, this feature is coming very soon!"
                 )
             return self.braket_gate(theta)
+        if language == Language.QASM2:
+            from mpqp.qasm.mpqp_to_qasm import float_to_qasm_str
+
+            instruction_str = self.qasm2_gate
+            if isinstance(self, (Rk, CRk)):
+                instruction_str += (
+                    "("
+                    + float_to_qasm_str(2 * np.pi / (2 ** float(self.parameters[0])))
```

Comment:
couldn't this be `float_to_qasm_str(self.theta)` instead ?

Reply:
done

Reply:
I replaced it with :
`instruction_str += f"({float_to_qasm_str(2 * np.pi / (2 ** float(self.k)))})"`
but I cannot replace it with theta, since theta is :
`return pi / 2 ** (self.k - 1)`